test/mockdata/mock_appdata.py

```diff
@@ -0,0 +1,14 @@
+from Testing import get_tested_mock_package
+
+APPDATA = get_tested_mock_package(
+files={
+'/usr/share/appdata/broken.appdata.xml': {'content_path': '/usr/share/appdata/broken.appdata.xml'},
```

Comment:
The correct path to use after move files to `test/files` will be:
```
'/usr/share/appdata/broken.appdata.xml': {'content-path': 'files/broken.appdata.xml'},
```

And the correct key is `content-path` not `content_path`, that's why even with the full system path this is working.